R/root.R

```diff
@@ -140,7 +139,7 @@ empty_config_contents <- function() {
 ## * also check that the outpack and orderly path are compatibible
 ##   (this is actually quite hard to get right, but should be done
 ##   before anything is created I think)
-root_open <- function(path, locate, require_orderly = FALSE, call = NULL) {
+root_open <- function(path, require_orderly = FALSE, call = parent.frame()) {
```

Comment:
Seems you've added values for `require_orderly` everywhere - would it be worth removing the default?